build.sbt

```diff
@@ -1,41 +1,61 @@
-import com.typesafe.sbt.SbtScalariform._
+import scalariform.formatter.preferences._
+import com.typesafe.sbt.SbtScalariform.ScalariformKeys
+import org.scalastyle.sbt.ScalastylePlugin
 
 name := "amqp-client-provider"
 
-version := "1.0.1"
+version := "1.1.0-SNAPSHOT"
```

Comment:
I think this change is backwards incompatible and therefore warrants a major version bump.